openapi/api.yml

```diff
@@ -1958,6 +1958,10 @@ definitions:
     title: PayRequest
   Payment:
     type: object
+    required:
```

Comment:
Maybe we should check if such change has an effect on autogenerated `Payment.decoder()` .. I would expect it to fail if the payment doesn't have the required fields

Reply:
Ok,  these three fields should be required (for the api that io-pay uses). If this is true, is this assumption fine? It's correct  update spec?